src/core/opentelemetry.ml

```diff
@@ -1576,8 +1576,9 @@ module Trace_context = struct
       | Ok sp -> Ok (Span_ctx.trace_id sp, Span_ctx.parent_id sp)
       | Error _ as e -> e
 
-    let to_value ~(trace_id : Trace_id.t) ~(parent_id : Span_id.t) () : string =
-      let span_ctx = Span_ctx.make ~trace_id ~parent_id () in
```

Comment:
Out of curiosity, what was the reason for this last `unit` argument? Is it because you anticipated this change and didn't want to break the interface ?

Reply:
Yes, when all arguments are labelled or optional, a `()` is useful to clearly mark full applications. And yes it allows adding optional arguments without breaking existing code :)